website/docs/r/event_orchestration_service.html.markdown

```diff
@@ -0,0 +1,202 @@
+* `expression`- (Required) A [PCL condition](https://developer.pagerduty.com/docs/ZG9jOjM1NTE0MDc0-pcl-overview) string.
+
+### Actions (`actions`) supports the following:
+* `route_to` - (Required) The ID of the target Service for the resulting alert.
+* `suppress` - (Optional) Set whether the resulting alert is suppressed. Suppressed alerts will not trigger an incident.
+* `suspend` - (Optional) The number of seconds to suspend the resulting alert before triggering. This effectively pauses incident notifications. If a `resolve` event arrives before the alert triggers then PagerDuty won't create an incident for this the resulting alert.
+* `priority` - (Optional) The ID of the priority you want to set on resulting incident. You can find the list of priority IDs for your account by calling the priorities endpoint.
+* `annotate` - (Optional) Add this text as a note on the resulting incident.
+* `pagerduty_automation_actions` - (Optional) Configure a [Process Automation](https://support.pagerduty.com/docs/event-orchestration#process-automation) associated with the resulting incident.
+  * `action_id` - (Required) Id of the Process Automation action to be triggered.
+* `automation_actions` - (Optional) Create a [Webhook](https://support.pagerduty.com/docs/event-orchestration#webhooks) associated with the resulting incident.
+  * `name` - (Required) Name of this Webhook.
+  * `url` - (Required) The API endpoint where PagerDuty's servers will send the webhook request.
+  * `auto_send` - (Optional) When true, PagerDuty's servers will automatically send this webhook request as soon as the resulting incident is created. When false, your incident responder will be able to manually trigger the Webhook via the PagerDuty website and mobile app.
+  * `headers` - (Optional) Specify custom key/value pairs that'll be sent with the webhook request as request headers.
+    * `key` - (Required) Name to identify the header
+    * `value` - (Required) Value of this header
+  * `parameters` - (Optional) Specify custom key/value pairs that'll be included in the webhook request's JSON payload.
+    * `key` - (Required) Name to identify the parameter
+    * `value` - (Required) Value of this parameter
+* `severity` - (Optional) sets Severity of the resulting alert. Allowed values are: `info`, `error`, `warning`, `critical`
+* `event_action` - (Optional) sets whether the resulting alert status is trigger or resolve. Allowed values are: `trigger`, `resolve`
+* `variables` - (Optional) Populate variables from event payloads and use those variables in other event actions.
+  * `name` - (Required) The name of the variable
+  * `path` - (Required) Path to a field in an event, in dot-notation. This supports both PagerDuty Common Event Format [PD-CEF](https://support.pagerduty.com/docs/pd-cef) and non-CEF fields. Eg: Use `event.summary` for the `summary` CEF field. Use raw_event.fieldname to read from the original event `fieldname` data.
```

Comment:
**Not a blocker:** I love that we're giving some examples of paths here. I also wonder if we could/should link to the PCL Path documentation here: https://developer.pagerduty.com/docs/ZG9jOjM1NTE0MDc0-pcl-overview#paths